scripts/projects.js

```diff
@@ -21,14 +19,20 @@ Project.prototype.toHtml = function(){
   return templateRender(this);
 }
 
-projectsInfo.forEach(function(projectObject) {
-  allProjects.push(new Project(projectObject));
-});
-
-allProjects.forEach(function(a){
-  $('#projects-container').append(a.toHtml());
-});
+function handelShowProjects(){
+  $.getJSON('/Data/projectsData.json')
+  .then(function(data){
+    data.forEach(function(projectsData){
+      allProjects.push(new Project(projectsData));
+    })
+    allProjects.forEach(function(a){
+      $('#projects-container').append(a.toHtml());
+    });
+  }, function(err) {
```

Comment:
A suggestion that inside this function after Line 32, you can put the following:

```
console.log(err)
```

So that in case an error occurs, it'll print to your console.